specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json

```diff
@@ -3413,11 +3413,11 @@
         },
         "type": {
           "type": "string",
```

Comment:
Unfortunately the list of extension types is not static or versioned (at least, not in any user-specifiable way); so I don't think an enum would work for it (I *wish* it were versioned :))